featuretools/tests/primitive_tests/aggregation_primitive_tests/test_agg_primitives.py

```diff
@@ -709,6 +718,391 @@ def test_ninf(self):
 
     def test_with_featuretools(self, es):
         transform, aggregation = find_applicable_primitives(self.primitive)
-        primitive_instantiate = self.primitive()
-        aggregation.append(primitive_instantiate)
+        primitive_instance = self.primitive()
+        aggregation.append(primitive_instance)
+        valid_dfs(es, aggregation, transform, self.primitive.name.upper())
+
+
+class TestMaxCount(PrimitiveTestBase):
+    primitive = MaxCount
+
+    def test_nan(self):
+        data = pd.Series([np.nan, np.nan, np.nan])
+        primitive_func = self.primitive().get_function()
+        answer = primitive_func(data)
+        assert np.isnan(answer)
+
+    def test_inf(self):
+        data = pd.Series([5, 10, 10, np.inf, np.inf, np.inf])
+        primitive_func = self.primitive().get_function()
+        answer = primitive_func(data)
+        assert answer == 3
+
+    def test_regular(self):
+        data = pd.Series([1, 1, 2, 3, 4, 4, 4, 5])
+        primitive_func = self.primitive().get_function()
+        answer = primitive_func(data)
+        assert answer == 1
+
+        data = pd.Series([1, 1, 2, 3, 4, 4, 4])
+        primitive_func = self.primitive().get_function()
+        answer = primitive_func(data)
+        assert answer == 3
+
+    def test_skipna(self):
+        data = pd.Series([1, 1, 2, 3, 4, 4, np.nan, 5])
```

Comment:
include True version here as well?

Reply:
True is the default, so it is tested in other tests